pom.xml

```diff
@@ -18,25 +18,30 @@
             <url>http://www.opensource.org/licenses/mit-license.php</url>
         </license>
     </licenses>
+
     <scm>
         <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
     </scm>
+
     <repositories>
         <repository>
             <id>repo.jenkins-ci.org</id>
             <url>http://repo.jenkins-ci.org/public/</url>
         </repository>
     </repositories>
+
     <pluginRepositories>
         <pluginRepository>
             <id>repo.jenkins-ci.org</id>
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+
     <properties>
-        <!-- TODO until bc4c132 in 1.545, or below 1.536 -->
-        <concurrency>1</concurrency>
+        <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
It's because a possible NPE here: 
return Jenkins.getInstance().getDescriptorByType(Whitelist.class);

I think it's something that is not gonna happen.